test/functional/rpc_deprecated.py

```diff
@@ -7,23 +7,22 @@
 
 class DeprecatedRpcTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 2
+        self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [[], ['-deprecatedrpc=bumpfee']]
+        self.extra_args = [[]]
 
     def run_test(self):
-        # This test should be used to verify correct behaviour of deprecated
-        # RPC methods with and without the -deprecatedrpc flags. For example:
+        # This test should be used to verify the errors of the currently
+        # deprecated RPC methods (without the -deprecatedrpc flag) until
+        # such RPCs are fully removed.
```

Comment:
Missed the `For example` part. In case retouched, fine otherwise.

Reply:
Added it back now